packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -396,62 +398,6 @@ describe('authentication', () => {
     });
   });
 
-  ['android', 'ios', 'desktop'].forEach(hostClientType => {
-    it(`should successfully pop up the auth window in the ${hostClientType} client`, async () => {
-      await utils.initializeWithContext('content', hostClientType);
-
-      const authenticationParams = {
-        url: 'https://someUrl',
-        width: 100,
-        height: 200,
-      };
-      authentication.authenticate(authenticationParams);
-
-      const message = utils.findMessageByFunc('authentication.authenticate');
-      expect(message).not.toBeNull();
-      expect(message.args.length).toBe(3);
-      expect(message.args[0]).toBe(authenticationParams.url.toLowerCase() + '/');
-      expect(message.args[1]).toBe(authenticationParams.width);
-      expect(message.args[2]).toBe(authenticationParams.height);
-    });
-
-    it(`should successfully handle auth success in the ${hostClientType} client`, async () => {
-      await utils.initializeWithContext('content', hostClientType);
-
-      const authenticationParams = {
-        url: 'https://someUrl',
-        width: 100,
-        height: 200,
-      };
-      const promise = authentication.authenticate(authenticationParams);
-
-      const message = utils.findMessageByFunc('authentication.authenticate');
-      expect(message).not.toBeNull();
-
-      utils.respondToMessage(message, true, 'someResult');
-
-      expect(promise).resolves.toEqual('someResult');
-    });
-
-    it(`should successfully handle auth failure in the ${hostClientType} client`, async () => {
-      await utils.initializeWithContext('content', hostClientType);
-
-      const authenticationParams = {
-        url: 'https://someUrl',
-        width: 100,
-        height: 200,
-      };
-      const promise = authentication.authenticate(authenticationParams);
-
-      const message = utils.findMessageByFunc('authentication.authenticate');
-      expect(message).not.toBeNull();
-
-      utils.respondToMessage(message, false, 'someReason');
-
-      return expect(promise).rejects.toThrowError('someReason');
```

Comment:
I don't see these tests removed in the original change, why do we need to remove them?

Reply:
These were discovered as duplicates. I knew I fixed these tests and thought it was weird they were failing, so I looked and there were two of each.

Reply:
The ones kept can be seen at line 332